src/components/header.js

```diff
@@ -38,46 +38,63 @@ const Header = () => {
   const [isOpen2, setIsOpen2] = useState(false);
 
   /* first dropdown items */
+
+  // TODO: was trying to have the links be dynamically created instead of hard-coding but whenever I tried to assign the length it resulted in undefined after the 2nd refresh
+  // const testMenuItemsOneLinks = t('header.nav.dropdown')[0]['simpleLink'];
+  // const testMenuItemsOneLength = testMenuItemsOneLinks.length; // assigning results in undefined
+  // console.log(testMenuItemsOneLinks);
+  // console.log(testMenuItemsOneLength);
+  // const testMenuItemsOneLength = t('header.nav.dropdown')[0]['simpleLink'].length; // assigning results in undefined
+  // const testMenuItemsOne = Array.from({ length: testMenuItemsOneLength }, (_, i) => {
+  //     return (
+  //       <Link to={"link" + i} key={i}>
+  //         {testMenuItemsOneLinks[i]}
+  //       </Link>
+  //     );
+  //   });
+
+  /* TODO: try to dynamically create */
   const testMenuItemsOne = [
-    <Link to="#linkOne" key="one">
-      {t("nav.dropdownOne.simpleLinkOne")}
+    <Link to="linkOne" key="one">
+      {t("header.nav.dropdown.0.simpleLink.0")}
```

Comment:
I feel like this syntax is not the most readable and also wouldn't be easy to update if more links were added in the dropdown array. Can you map through the dropdown array to create the menu items, instead of hardcoding the number of menu items?

Reply:
Yes, that's the goal. But I'm having the same trouble dynamically creating the links as I am on line 42. It's something I may take into account next week.

Reply:
Mapped through the `dropdowns` and `parentLinks` arrays in the translation files to create parent links and the menu items for each dropdown in the navigation. Further refactoring will be done by introducing a `NavDropDown` component in the next PR.